src/Writer/FlushableWriter.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Ddeboer\DataImport\Writer;
+
+interface FlushableWriter
```

Comment:
Because this is a quite different approach. A finish method should be called at the end of the workflow (e.g. close a database connection or call `fclose`) and the `flush` method means, write a anything to the database/file/webservice.